### DISCUSSION TITLE
Alter notice?

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#17166] [Archive]
Why cant we alter a field type "notice". I think this would be something very helpfull.

**Comment by paescuj** [comment]
Maybe I don't understand it correctly, but you can change the text as follows:
* Create field with some text:
  <img width="300" alt="Screenshot 2023-01-15 at 00 16 45" src="https://user-images.githubusercontent.com/5363448/212501209-b1c39994-78bd-42fc-9091-89175eaa298b.png">
* Click on "Edit Field" -> "Interfaces" and change the text:
  <img width="300" alt="Screenshot 2023-01-15 at 00 16 59" src="https://user-images.githubusercontent.com/5363448/212501211-ea930c9c-bb53-4720-9fd2-55fc942e5028.png">
